content/en/docs/concepts/architecture/nodes.md

```diff
@@ -570,6 +574,9 @@ see the blog-post about [Kubernetes 1.28: NodeSwap graduates to Beta1](/blog/202
 [KEP-2400](https://github.com/kubernetes/enhancements/issues/4128) and its
 [design proposal](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2400-node-swap/README.md).
 
+<!-- Below shortcode creates a "API reference" section linking to the relevant API reference for this page. -->
+{{< api-reference-link >}}
+
 ## {{% heading "whatsnext" %}}
```

Comment:
I'd put the link inside the further reading, eg:

```suggestion
## {{% heading "whatsnext" %}}

- {{% page-api-reference %}}
```

and possibly with the option to do:
```suggestion
## {{% heading "whatsnext" %}}

Read about the Node API:
- {{% page-api-reference kind="Node" %}}
```

(in case we need to be explicit about what we're linking to)